joomla/joomla-cms
# Make JLanguageHelper::getInstalledLanguages add log message when language xml files doesn't exist (or is not readable)

Pull Request for New Issue.

### Summary of Changes

When the language xml files does not exist the new `JLanguageHelper::getInstalledLanguages` is returning an exception. This could cause some B/C issues.
So this PR makes it fail silently instead.

### Testing Instructions

1. Have a site with 2 or more languages installed
2. Delete (or rename) one site language xml file (in `/languages/xx-XX/xx-XX.xml`)
3. Go to Extensions -> Languages -> Installed. You will see an exception being trowed
4. Apply patch
5. Repeat step 3, no exception and you will see a log message in debug console

### Documentation Changes Required

None.
